backend/apps/objects/views.py

```
@@ -89,32 +100,47 @@ def delete(self, request, object_id):
+                rental = ObjectRental.objects.create(
+                    object=obj,
+                    user=request.user,
+                    start_date=start,
+                    end_date=end,
+                )
             return JsonResponse({"id": rental.id, "detail": "Reserva creada."}, status=201)
         except IntegrityError:
             return JsonResponse({"detail": "Ya tienes una reserva conflictiva."}, status=400)
+        except json.JSONDecodeError:
+            return JsonResponse({"detail": "El cuerpo de la petición no es JSON válido."}, status=400)
         except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)